node/_fs/_fs_unlink.ts

```diff
@@ -4,5 +4,12 @@ export function unlink(path: string | URL, callback: (err?: Error) => void) {
 }
 
 export function unlinkSync(path: string | URL) {
-  Deno.removeSync(path);
+  try {
+    Deno.removeSync(path);
+  } catch (e) {
+    if (e instanceof Deno.errors.NotFound) {
+      e.code = "ENOENT";
+    }
+    throw e;
+  }
```

Comment:
This is the last bit needed to get ESLint working

Reply:
Ref https://github.com/denoland/deno/pull/12591

Reply:
@bartlomieju Can you try the latest version of https://github.com/denoland/deno/pull/12591 ?

Reply:
It works with that PR